### PR TITLE
MLT-0073 Define categories

### PR DESCRIPTION
**Pull request by anotheroneofthese**
This PR implements concrete Item Categories to use within Hermes WLS

### CODE REVIEW

File: src/main/kotlin/no/nb/mlt/wls/domain/model/ItemCategory.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package no.nb.mlt.wls.domain.model
+
+enum class ItemCategory {
+    PAPER,
+    DISC,
+    FILM,
+    PHOTO,
+    EQUIPMENT,
```
Comment (MormonJesus69420) [inline src/main/kotlin/no/nb/mlt/wls/domain/model/ItemCategory.kt:8]: Need to consider how we want to name this category, "gjenstand" translates pretty much into object, but that's not fitting. Equipment is a possibility; other names we could give it:

- hardware
- appliance
- resource
- gadget
- apparatus
- paraphernalia
- accoutrement
- appurtenance
- "bits and bobs" (could also rename bulk items to "odds and sods")

File: src/main/kotlin/no/nb/mlt/wls/application/hostapi/item/ApiItemPayload.kt [hunk 2]
```diff
@@ -47,9 +48,12 @@ data class ApiItemPayload(
     val description: String,
     @Schema(
         description = "What kind of item category the item belongs to, e.g. Books, Issues, Films, etc.",
-        examples = ["BOOK", "ISSUE", "Arkivmateriale", "Film_Frys"]
+        examples = [
+            "safetyfilm", "nitratfilm", "film", "plater", "fotografier", "papir",
+            "gjenstand", "lydbånd", "videobånd", "sekkepost", "magnetbånd"
```
Comment (MormonJesus69420) [inline src/main/kotlin/no/nb/mlt/wls/application/hostapi/item/ApiItemPayload.kt:53]: Will fix it myself, but the examples ain't right

File: src/main/kotlin/no/nb/mlt/wls/infrastructure/callbacks/NotificationItemPayload.kt [hunk 3]
```diff
@@ -28,9 +29,12 @@ data class NotificationItemPayload(
     val description: String,
     @Schema(
         description = "What kind of item category the item belongs to, e.g. Books, Issues, Films, etc.",
-        examples = ["BOOK", "ISSUE", "Arkivmateriale", "Film_Frys"]
+        examples = [
+            "safetyfilm", "nitratfilm", "film", "plater", "fotografier", "papir",
+            "gjenstand", "lydbånd", "videobånd", "sekkepost", "magnetbånd"
```
Comment (MormonJesus69420) [inline src/main/kotlin/no/nb/mlt/wls/infrastructure/callbacks/NotificationItemPayload.kt:34]: Same as before

File: src/test/kotlin/no/nb/mlt/wls/item/model/ItemModelValidationTest.kt [hunk 4]
```diff
@@ -56,17 +57,18 @@ class ItemModelValidationTest {
...
+//    fun `item with blank itemCategory should fail validation`() {
+//        val item = validItem.copy(itemCategory = "")
+//
+//        val thrown = catchThrowable(item::validate)
+//
+//        then(thrown)
+//            .isNotNull()
+//            .isInstanceOf(ValidationException::class.java)
+//            .hasMessageContaining("category")
+//    }
```
Comment (MormonJesus69420) [inline src/test/kotlin/no/nb/mlt/wls/item/model/ItemModelValidationTest.kt:71]: Yas queen, yeet!

**Review by MormonJesus69420 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/no/nb/mlt/wls/domain/model/ItemCategory.kt (hunk 1 above)
Comment (anotheroneofthese) [reply src/main/kotlin/no/nb/mlt/wls/domain/model/ItemCategory.kt:8]: As discussed this remains EQUIPMENT for now. Odds and sods unfortunately did not make this years pageant, but maybe next year

**Review by MormonJesus69420 — APPROVED** [review]

**Pull request merged**